FileMapper/pickerPrompt_script.html

```diff
@@ -0,0 +1,215 @@
+   * fails to load
+   */
+  function displayPickerFailureAlert() {
+    var message =
+        'Unable to load the file picker. <br> <strong>Please try again after some time.</strong>';
+    showError(message);
+  }
+
+  /**
+   * Function to check if the localPathInput value is marked invalid
+   * before loading the file picker
+   */
+  function localPathInputError() {
+    var localPathInput = document.getElementById("localpathinput");
+    if (localPathInput.style.borderColor === 'red') {
+      showError(
+          "The local path provided is invalid. <br> <strong> Please provide a valid local path and then try again. </strong>");
+      return;
+    }
+    // If there is no error in the local path then we can show the picker
+    loadPicker();
+  }
+
+  /**
+   * A callback function that extracts the chosen document's metadata from the
+   * response object. For details on the response object, see
+   * https://developers.google.com/picker/docs/result
+   *
+   * @param {object} data The response object.
+   */
+  function pickerCallback(data) {
+    var action = data[google.picker.Response.ACTION];
+    if (action == google.picker.Action.PICKED) {
+      var doc = data[google.picker.Response.DOCUMENTS][0];
+      var id = doc[google.picker.Document.ID];
+      var url = doc[google.picker.Document.URL];
+      var mimetype = doc[google.picker.Document.MIME_TYPE];
+      var title = doc[google.picker.Document.NAME];
+
+      showError(
+          "<strong>" + title +
+          "</strong> is selected from Google Drive to map to <strong>" +
+          document.getElementById("localpathinput").value +
+          "</strong> <br> <strong> Mapping is being added to the Config.<strong>");
+
+      var isFolder =
+          (mimetype === "application/vnd.google-apps.folder") ? true : false;
+
+      google.script.run
+          .withSuccessHandler(function() {
+            showError(
+                "Mapping is added successfully to the config.<br><strong>Now, the VBA macro execution will resume.</strong>");
+            setTimeout(function() { google.script.host.close(); }, 1000);
+          })
+          .withFailureHandler(function() {
+            showError(
+                "Failed to add the mapping to the config. <br> <strong>Please try again.</strong>");
+          })
+          .processPickedFile(id, isFolder,
+                             document.getElementById("localpathinput").value);
+    } else if (action == google.picker.Action.CANCEL) {
+      showError(
+          "Picker was closed without selecting a file. <br> <strong>Please try again.</strong>");
+    }
+  }
+
+  /**
+   * Function to return the file picker view based on the file extension
+   *
+   * @param {String} extension The extension of the local path
+   * @return {View} View in Google picker for the MimeType of the file based on
+   *     it's extension
+   */
+  function createViewUsingExtension(extension) {
```

Comment:
This block is a copy of the one in other CL? It's hard to figure out if it's the same file in 2 different PRs

Reply:
No, this is a separate block. This block is to decide the view of the file picker based on the file extensions. 
I don't have same files in 2 different PRs.
Also, for the update that i made in the mimetypes, i will make here in the views as well, couldn't find a way to do that yesterday but now i have a possible way out. Will update this.